code/name-service/resolve-sol-domain/resolve-sol-domain.en.ts

```diff
@@ -1,27 +1,23 @@
 import {
-    getHashedName,
-    getNameAccountKey,
-    NameRegistryState
-} from "@solana/spl-name-service"
```

Comment:
`@solana/spl-name-service` is outdated. `@bonfida/spl-name-service` is up to date and contains more utility functions